spring-cloud-dataflow-common/spring-cloud-dataflow-common-test-docker/src/test/java/org/springframework/cloud/dataflow/common/test/docker/compose/execution/DockerComposeTests.java

```diff
@@ -168,43 +168,43 @@ void throwExceptionWhenDownFailsForAReasonOtherThanTheCommandNotBeingPresent() {
     @Test
     void useTheRemoveVolumesFlagWhenDownExists() throws Exception {
         compose.down();
-        verify(executor).execute("down", "--volumes");
+        verify(executor).execute(true,"down", "--volumes");
```

Comment:
nitpick space between `true,` and `"down"`.  
Please check other tests I saw this in a couple other places.   Call me checkstyle :-D